### PR TITLE
Fixes #1141: Added wikipedia logo and improved design of infobox

### LINKED ISSUE #1141: Improve design of Infobox by using Wikipedia logo
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Expected behavior:**
Improve design of Infobox by using Wikipedia logo before `More at Wikipedia` link.
**Screenshots of the issue:**

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1141 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

- Added wikipedia logo and improved design of infobox

- https://pr-1142-fossasia-susper.surge.sh

image

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks. Please reduce the size of the Wikipedia logo a bit more.

File: src/assets/images/wikipedia-w-brands.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" data-prefix="fab" data-icon="wikipedia-w" class="svg-inline--fa fa-wikipedia-w fa-w-20" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512"><path fill="currentColor" d="M640 51.2l-.3 12.2c-28.1.8-45 15.8-55.8 40.3-25 57.8-103.3 240-155.3 358.6H415l-81.9-193.1c-32.5 63.6-68.3 130-99.2 193.1-.3.3-15 0-15-.3C172 352.3 122.8 243.4 75.8 133.4 64.4 106.7 26.4 63.4.2 63.7c0-3.1-.3-10-.3-14.2h161.9v13.9c-19.2 1.1-52.8 13.3-43.3 34.2 21.9 49.7 103.6 240.3 125.6 288.6 15-29.7 57.8-109.2 75.3-142.8-13.9-28.3-58.6-133.9-72.8-160-9.7-17.8-36.1-19.4-55.8-19.7V49.8l142.5.3v13.1c-19.4.6-38.1 7.8-29.4 26.1 18.9 40 30.6 68.1 48.1 104.7 5.6-10.8 34.7-69.4 48.1-100.8 8.9-20.6-3.9-28.6-38.6-29.4.3-3.6 0-10.3.3-13.6 44.4-.3 111.1-.3 123.1-.6v13.6c-22.5.8-45.8 12.8-58.1 31.7l-59.2 122.8c6.4 16.1 63.3 142.8 69.2 156.7L559.2 91.8c-8.6-23.1-36.4-28.1-47.2-28.3V49.6l127.8 1.1.2.5z"></path></svg>
```
Comment (simsausaurabh) [inline src/assets/images/wikipedia-w-brands.svg]: Please add a new line at EOF. Rest looks good :+1:

Comment (praveenojha33) [reply src/assets/images/wikipedia-w-brands.svg]: @simsausaurabh Done :+1:

**Comment by mariobehling** [comment]
The Wikipedia logo is not aligned with the text. It also does not provide any additional feature that we really need. Please take out the Wikipedia logo and follow Google.

**Comment by praveenojha33** [comment]
@mariobehling Ok sure I am reverting it.

**Pull request merged**